### PR TITLE
Add logo and reformat

### PR DESCRIPTION
**Pull request by daviortega**

### CODE REVIEW

**Comment by daviortega** [comment]
Hello @metacoin !!! :)

File: README.md [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-## Q. When was FLO created and launched?
-## A. Flow was created and launched in 2013. It is one of the oldest surviving cyrptocurrencities. Its Genesis Block was mined on    June 17, 2013.
+# FLO - FAQ
+Community driven FAQ for the FLO blockchain project - send us a pull request with questions or answers!
 
-## Q. I've heard FLO was established via a "fair launch." What does that mean?
-## A. A "fair launch" means there was no ICO, no premine, and no founders’ reward.
+### Q. When was FLO created and launched?
+_last update: May 31st 2019_  
+A. Flow was created and launched in 2013. It is one of the oldest surviving cyrptocurrencities. Its Genesis Block was mined on    June 17, 2013.
```
Comment (bitspill) [inline README.md]: Working from a mobile device? Seems autocorrect strikes again.

Comment (jasonschmitz) [reply README.md]: Oops. Will fix

**Review by metacoin — APPROVED** [review]
Looks good, thank you

File: README.md (hunk 1 above)
Comment (metacoin) [reply README.md]: I got it

**Pull request merged**